ee/clickhouse/queries/clickhouse_trends.py

```diff
@@ -1,10 +1,398 @@
+            if filter.breakdown:
+                extra_label = self._determine_breakdown_label(filter.breakdown_type, filter.breakdown[idx])
+                label = "{} - {}".format(entity.name, extra_label)
+                additional_values = {"label": label, "breakdown_value": filter.breakdown[idx]}
+            else:
+                additional_values = {}
+            counts = stats[1]
+            dates = [
+                ((item - timedelta(days=1)) if filter.interval == "month" else item).strftime(
+                    "%Y-%m-%d{}".format(", %H:%M" if filter.interval == "hour" or filter.interval == "minute" else "")
+                )
+                for item in stats[0]
+            ]
+            labels = [
+                ((item - timedelta(days=1)) if filter.interval == "month" else item).strftime(
+                    "%a. %-d %B{}".format(", %H:%M" if filter.interval == "hour" or filter.interval == "minute" else "")
+                )
+                for item in stats[0]
+            ]
+            parsed.append({"data": counts, "count": sum(counts), "dates": dates, "labels": labels, **additional_values})
+
+        return parsed
+
+    def _determine_breakdown_label(
+        self, breakdown_type: Optional[str], breakdown: Optional[Union[str, int]] = ""
+    ) -> str:
+        if breakdown_type == "cohort":
+            if breakdown == "all":
+                return "all users"
+            else:
+                return Cohort.objects.get(pk=breakdown).name
+        elif breakdown_type == "person":
+            return ""
+        else:
+            return str(breakdown) or ""
+
+    def _format_normal_query(self, entity: Entity, filter: Filter, team: Team) -> List[Dict[str, Any]]:
+
+        inteval_annotation = get_interval_annotation_ch(filter.interval)
+        num_intervals, seconds_in_interval = get_time_diff(filter.interval or "day", filter.date_from, filter.date_to)
+        parsed_date_from, parsed_date_to = parse_timestamps(filter=filter)
+        prop_filters, prop_filter_params = parse_prop_clauses("id", filter.properties, team)
+
+        params: Dict = {"team_id": team.pk}
+        params = {**params, **prop_filter_params}
+        if entity.type == TREND_FILTER_TYPE_ACTIONS:
+            try:
+                action = Action.objects.get(pk=entity.id)
+                action_query, action_params = format_action_filter(action)
+                params = {**params, **action_params}
+                content_sql = VOLUME_ACTIONS_SQL.format(
+                    interval=inteval_annotation,
+                    timestamp="timestamp",
+                    team_id=team.pk,
+                    actions_query=action_query,
+                    parsed_date_from=(parsed_date_from or ""),
+                    parsed_date_to=(parsed_date_to or ""),
+                    filters="{filters}".format(filters=prop_filters) if filter.properties else "",
+                )
+            except:
+                return []
```

Comment:
It feels a bit risky to just return `[]` on any error.

In the sticky branch I had a similar situation and solved it with:

```
            if action_query == "":
                return []
```

after line 330 (`action_query, action_params = format_action_filter(action)`)

This way if there's any real and unexpected error, we'll still get a bug report.